sdk_container/.repo/manifests/mantle-container

```diff
@@ -1 +1 @@
-ghcr.io/flatcar/mantle:git-790b16559be109a124536110a6841251d7cacb71
+ghcr.io/flatcar/mantle:git-20a2f8ffee8c8a1a042b1da99f0f59312110f285
```

Comment:
This ref is outdated, `d638477431717aae4d87909d5ee5e39f009a803f` is the latest as of now. I merged the mantle bumps and your changes are in, so you can drop this change here.

Reply:
Rebased on latest main, so this got pulled in automatically. I made sure the ref is correct though.